cgd_tx_eff/src/edu/ohsu/compbio/annovar/annovar_parser.py

```diff
@@ -422,14 +425,14 @@ def parse_file(self, annovar_file_type: AnnovarFileType, file_name:str, delimite
             for row in reader:
                 if annovar_file_type == AnnovarFileType.ExonicVariantFunction:
                     assert row[0].startswith('line')
-                    if len(row) != 18:
-                        self.logger.info(f"Exonic variant function file is expected have 18 columns. Found {len(row)}. Make sure you run the annotate_variation.pl script with the '--otherinfo' parameter to include other information from the original VCF")
+                    if len(row) != 9:
+                        self.logger.info(f"Exonic variant function file is expected have 9 columns. Found {len(row)}.")
                     
                     # Parse a row in an exonic_variant_function file
                     annovar_recs.extend(self._parse_exonic_variant_function_row(row))
                 elif annovar_file_type == AnnovarFileType.VariantFunction:
-                    if len(row) != 17:
-                        self.logger.info(f"Variant function file file is expected to have 17 columns. Found {len(row)}. Make sure you run the annotate_variation.pl with the '--otherinfo' parameter to include other information from the original VCF")
+                    if len(row) != 8:
+                        self.logger.info(f"Variant function file is expected to have 17 columns. Found {len(row)}. Make sure you run the annotate_variation.pl with the '--otherinfo' parameter to include other information from the original VCF")
```

Comment:
Should be `expected to have 8 columns`. The subsequent part about the VCF is out of date, too.